CHANGELOG.md

```diff
@@ -1,3 +1,208 @@
+* [#287:](https://github.com/pysal/momepy/issues/287) momepy.enclosures doesn't appear to observe limit parameter 
+* [#270:](https://github.com/pysal/momepy/issues/270) Enclosure has problems with rivers 
+* [#291:](https://github.com/pysal/momepy/pull/291) BUG: Tessellation error on non-standard enclosures 
+* [#289:](https://github.com/pysal/momepy/issues/289) BUG: momepy.Tessellation fails with IndexError 
+* [#290:](https://github.com/pysal/momepy/pull/290) small typo in gdf_to_nx 
+
+
+<a name="spglm"></a>
+### spglm
+
+
+<a name="spint"></a>
+### spint
+
+
+<a name="spreg"></a>
+### spreg
+
+
+<a name="spvcm"></a>
+### spvcm
+
+
+<a name="tobler"></a>
+### tobler
+
+
+<a name="mapclassify"></a>
+### mapclassify
+
+
+<a name="splot"></a>
+### splot
+
+
+<a name="spopt"></a>
+### spopt
+* [#205:](https://github.com/pysal/spopt/pull/205) install pulp for docs build 
+* [#196:](https://github.com/pysal/spopt/issues/196) rework JOSS manuscript 
+* [#200:](https://github.com/pysal/spopt/pull/200) [WIP] JOSS paper revisions work 
+* [#203:](https://github.com/pysal/spopt/issues/203) update the real-world facility location notebook. 
+* [#204:](https://github.com/pysal/spopt/pull/204) Update fac-loc real world notebook 
+* [#201:](https://github.com/pysal/spopt/issues/201) region_k_means not handling candidate move updates 
+* [#202:](https://github.com/pysal/spopt/pull/202) Update candidates list after making a move 
+* [#179:](https://github.com/pysal/spopt/issues/179) Update changelog tooling to report contributors 
+* [#199:](https://github.com/pysal/spopt/pull/199) Generate Changelog notes automatically on release 
+* [#197:](https://github.com/pysal/spopt/pull/197) access package version in tools/gitcount 
+* [#198:](https://github.com/pysal/spopt/pull/198) Add examples section to locate module 
+* [#190:](https://github.com/pysal/spopt/issues/190) update locate docs 
+* [#194:](https://github.com/pysal/spopt/pull/194) switch to versioneer 
+* [#193:](https://github.com/pysal/spopt/pull/193) Update locate docs 
+* [#192:](https://github.com/pysal/spopt/issues/192) Reference error for Church and Murray book 
+* [#2:](https://github.com/pysal/spopt/issues/2) facility location models in a class 
+* [#189:](https://github.com/pysal/spopt/issues/189) Facility Location modeling solutions & CI 
+* [#186:](https://github.com/pysal/spopt/pull/186) Add Locate Module 
+* [#3:](https://github.com/pysal/spopt/issues/3) bug: maxp implementation is not using contiguity 
+* [#49:](https://github.com/pysal/spopt/issues/49) spopt as solver agnostic? 
+* [#61:](https://github.com/pysal/spopt/issues/61) solvers and solver APIs 
+* [#18:](https://github.com/pysal/spopt/issues/18) new alternative to pulp 
+
+<a name="contributors"></a>
+## Contributors
+
+Many thanks to all of the following individuals who contributed to this release:
+
+
+ - Charles Schmidt
+ - Eli Knaap
+ - Elliott Sales De Andrade
+ - Gegen07
```

Comment:
The @gegen07 GH handle is Germano Barcelos. Should we remove the GH handle in favor of only his name or keep both?